<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWidget</class>
 <widget class="QWidget" name="TableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="listView">
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>添加餐桌</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButton_add">
          <property name="text">
           <string>点菜</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_delete">
          <property name="text">
           <string>退菜</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>客人数量：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>餐桌：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>菜名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>单价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>208号台</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_count">
            <property name="text">
             <string>0人</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>人</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>开台</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_total">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButton_4">
     <property name="text">
      <string>撤台</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
